src/parser.rs

```diff
@@ -178,6 +185,19 @@ pub fn parse(pairs: Pairs<Rule>) -> Result<Items, Error<Rule>> {
                         body,
                     })
                 }
+                Rule::table => {
+                    let mut items = parse(pair.into_inner())?.into_iter();
+                    let name = if let Item::Ident(ident) = items.next().unwrap() {
+                        ident
+                    } else {
+                        panic!("Expected Table, got {:?}", items)
```

Comment:
I think these should be unreachable — it shouldn't be possible to produce a `table` where the first item isn't an `ident`, or the second isn't a `pipeline`.

Though the practical difference isn't that high, please feel free to merge!

Reply:
OK! Thanks!